CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 Mapbox welcomes participation and contributions from everyone.
 
 # main
+## Bug fixes 🐞
+* Fix terrain rendering for the devices with GPUs that support OES_texture_float_linear extension but do not support OES_texture_float extension. ([2036](https://github.com/mapbox/mapbox-maps-android/pull/2036)) 
```

Comment:
nitpicking but perhaps simplify? fix terrain not being rendered on specific devices (like we did for v10.12.0-rc.1 changelog)

Reply:
Okay, this one was copied from gl-native, but agree to simplify for users